stdlib/public/runtime/Errors.cpp

```diff
@@ -69,7 +138,17 @@ reportOnCrash(const char *message)
 static void
 reportOnCrash(const char *message)
 {
-  // empty
+  fputs("Current stack trace:\n", stderr);
+  int count;
+  char **trace = reportBacktrace(&count);
+  for (int i = 0; i < count; i++) {
+#if SWIFT_HAVE_WORKING_STD_REGEX
+    std::cerr << demangledLine(trace[i]) << "\n";
+#else
+    std::cerr << trace[i] << "\n";
```

Comment:
Use fwrite or fputs here. C++ streams have launch time overhead that we want to avoid if we can.